cdm/core/src/main/java/ucar/nc2/internal/iosp/hdf4/H4iosp.java

```diff
@@ -64,11 +68,24 @@ public String getFileTypeDescription() {
   public void open(RandomAccessFile raf, NetcdfFile ncfile, CancelTask cancelTask) throws IOException {
     super.open(raf, ncfile, cancelTask);
     Group.Builder rootGroup = Group.builder(null).setName("").setNcfile(ncfile);
-    header.read(raf, rootGroup, null);
+    getHeader().read(raf, rootGroup, null);
     ncfile.setRootGroup(rootGroup.build(null));
     ncfile.finish();
   }
 
+  /**
+   * Return header for reading netcdf file.
```

Comment:
In this comment, this is the header for reading an HDF4 file, not a netCDF file.